packages/react/test/test-provider.spec.tsx

```diff
@@ -45,22 +45,21 @@ describe('OpenFeatureTestProvider', () => {
       const delay = 100;
       render(
         <OpenFeatureTestProvider delayMs={delay} flagValueMap={{ [FLAG_KEY]: true }}>
-          <TestComponent />
+          <TestComponent suspend={false} />
         </OpenFeatureTestProvider>,
       );
 
       // should only be resolved after delay
-      expect(await screen.findByText('👎')).toBeInTheDocument();
+      expect(screen.getByText('👎')).toBeInTheDocument();
```

Comment:
`getByText` is synchronous, which is much more reliable for these suspense-sensitive tests. I've updated all assertions to use this since `findByText` was hiding some bugs.